src/chunk_tag_t_keywords.h

```diff
@@ -0,0 +1,310 @@
+static chunk_tag_t keywords[] =
```

Comment:
We should add a #if define guard to the file, to avoid multiple definition in case the file gets included twice in the same compile unit.

Reply:
Yes